lib/paperclip/thumbnail.rb

```diff
@@ -76,7 +77,7 @@ def make
 
         parameters = parameters.flatten.compact.join(" ").strip.squeeze(" ")
 
-        success = convert(parameters, :source => "#{File.expand_path(src.path)}#{'[0]' unless animated?}", :dest => File.expand_path(dst.path))
+        success = convert(parameters, :source => "#{File.expand_path(src.path)}#{'[' + @frame_index.to_s + ']' unless animated?}", :dest => File.expand_path(dst.path))
```

Comment:
While it's true that existing code doesn't consistently adhere to the style guides we want to follow, but new code should whenever possible. Suggested rewrite:

```ruby
success = convert(
  parameters,
  source: "#{File.expand_path(src.path)}#{'[' + @frame_index.to_s + ']' unless animated?}",
  dest: File.expand_path(dst.path),
)
```

(I would even extract a nicer private method for `source:`'s value, but that's not needed now).